pkg/build/scripts/users/add-users.sh.tpl

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+# Without this, the script will run successfully during combustion, but when /home
+# is mounted it will hide the /home used during these user creations.
+mount /home
+
+{{ range . }}
+
+{{/* Non-root users */}}
+{{ if (ne .Username "root") }}
+  useradd -m {{.Username}}
+  {{ if .Password }}
+    echo '{{.Username}}:{{.Password}}' | chpasswd -e
+  {{ end }}
```

Comment:
If this is the same for both cases can we extract it down below in order to avoid the repetition?

Reply:
IMO that makes it less readable, but that may be a personal distaste for the templating syntax. I like the model here where we have distinct section on root v. non-root and, IMO, that's worth the minimal duplication.